app/models/setting.rb

```diff
@@ -2,9 +2,10 @@ class Setting < ActiveRecord::Base
   extend FriendlyId
   friendly_id :name
   include ActiveModel::Validations
+  include EncryptField
   self.inheritance_column = 'category'
 
-  TYPES= %w{ integer boolean hash array string }
+  TYPES= %w{ integer boolean hash array string encrypted }
```

Comment:
Why add this as an additional type instead of an extra flag?  It looks like we'd lose type-checking on any settings using this, while if it was a separate flag then we'd not lose the type information.